examples/playwright-browser-agent/fill_form_example.py

```
@@ -0,0 +1,103 @@
+        # Some sites (Google Forms in particular) ship fields as React-controlled
+        # inputs that start with `disabled` set and only enable once their init
+        # JS finishes loading. If a CDN the page needs is not in the egress
+        # whitelist, those inputs stay disabled forever — see README, Layer 1.
+        # Wait up to 15s for the first text input to enable; proceed regardless.
+        try:
+            page.wait_for_function(
+                "!document.querySelector('input[type=\"text\"]').disabled",
+                timeout=15000,
+            )
+            print("Fields enabled.")
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)